Dockerfile

```
@@ -0,0 +1,20 @@
+# Builder
+FROM node:alpine3.14 as builder
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY . ./
+RUN yarn install
+
+# Runner
+FROM node:alpine3.14
+
+USER node
+
+COPY --from=builder /app /app
+
+WORKDIR /app
+
+COPY entrypoint.sh /
+ENTRYPOINT [ "/entrypoint.sh" ]
```

Perhaps we can just do `yarn liquidator` instead of a separate shell script? 
```suggestion
ENTRYPOINT [ "/entrypoint.sh" ]
```
ENTRYPOINT [ "yarn", "liquidator" ]